mage_ai/data_integrations/utils/scheduler.py

```diff
@@ -79,6 +80,24 @@ def initialize_state_and_runs(
         raise err
 
 
+def start_scheduler(pipeline_run: PipelineRun) -> None:
```

Comment:
Let's put this method in pipeline_scheduler.py. Part of this method is not related to data integration pipeline.